projects/demo/src/app/demo/pages/modal-page/modal-page.component.ts

```diff
@@ -85,21 +85,24 @@ export class OwnModalComponent extends BaseModalComponent implements AfterViewIn
         this.pastanaga.modalService.openModal(DialogExampleComponent).onClose.subscribe(data => console.log('Modal closed', data));
     }
 }`;
-    closingProgrammatically = `export class SomeDialogComponent implements IModal {
+    closingProgrammatically = `export class SomeDialogComponent {
     closeDialog() {
         this.modal.close({whatever: true, answer: 42});
     }
 }`;
     passingDataToModal = `export class CallerComponent {
     open() {
-        const modalRef = this.pastanaga.modalService.openModal(SomeDialogComponent);
-        modalRef.data = { document: myDoc, user: myUser };
+        const modalRef = this.pastanaga.modalService.openModal(SomeDialogComponent, {
```

Comment:
sorry that was stuff I forgot to cleaned up when removing pastanaga service, but can you update all the code example to use `this.modalService` instead of `this.pastanaga.modalService` ?